src/webui/src/components/PackageSidebar/index.jsx

```diff
@@ -0,0 +1,192 @@
+      showAllContributors: _.size(packageMeta.latest.contributors) <= 5
+    });
+  }
+
+  showAllContributors() {
+    this.setState({
+      showAllContributors: true
+    });
+  }
+
+  render() {
+    let {author, contributors, recentReleases, lastUpdate, showAllContributors, dependencies} = this.state;
+
+    let uniqueContributors = [];
+    if (contributors) {
+      uniqueContributors = _.filter(_.uniqBy(contributors, (contributor) => contributor.name), (contributor) => {
+        return contributor.name !== _.get(author, 'name');
+      })
+        .slice(0, 5);
+    }
+
+    return (
+      <aside>
+        <Module
+          title="Last Sync"
+          description={lastUpdate}
+          className={classes.releasesModule}
+        >
+          <ul>
+            {recentReleases.length > 0 && recentReleases.map((versionInfo) => {
+              return (
+                <li key={versionInfo.version}>
+                  <span>{versionInfo.version}</span>
+                  <span>{(new Date(versionInfo.time)).toLocaleString()}</span>
+                </li>
+              );
+            })}
+          </ul>
+        </Module>
+
+        <Module
+          title="Maintainers"
+          className={classes.authorsModule}
+        >
+          <ul>
+            {author && <MaintainerInfo title="Author" name={author.name} avatar={author.avatar}/>}
+            {contributors && (showAllContributors ? contributors : uniqueContributors).map((contributor, index) => {
+              return <MaintainerInfo key={index} title="Contributors" name={contributor.name} avatar={contributor.avatar}/>;
+            })}
+          </ul>
+          {!this.state.showAllContributors && (
+            <button
+              onClick={this.showAllContributors}
+              className={classes.showAllContributors}
+              title="Current list only show the author and first 5 contributors unique by name"
+            >
+              Show all contributor
+            </button>
+          )}
+        </Module>
+
+        {/* Package management module? Help us implement it! */}
+
+        <Module
+          title="Dependencies"
+          className={classes.dependenciesModule}
+        >
+          <ul>
+            {_.size(dependencies) ? (
+              Object.keys(dependencies).map((dependenceName, index) => {
+                return (
+                  <li key={index} title={`Depend on version: ${dependencies[dependenceName]}`}>
+                    <a href={`${getRegistryURL()}/#/detail/${dependenceName}`} target="_blank">{dependenceName}</a>
+                    {index + 1 < _.size(dependencies) && <span>,&nbsp;</span>}
+                  </li>
+                );
+             })
+            ): <p className={classes.emptyPlaceholder}>Zero dependencies</p>}
+          </ul>
+        </Module>
+      </aside>
+    );
+  }
+}
+PackageSidebar.propTypes = {
+  packageName: PropTypes.string.isRequired
+};
+
+function Module(props) {
+  return (
```

Comment:
`<Module >` is a great idea, but as @ayusharma we can wrap the 3 sections and reuse module multiple times and reduce the `render` method size